docs/maintain-guides-validator-community.md

```diff
@@ -0,0 +1,275 @@
+distro-packaged libraries, any server hardening practices, etc.
+
+#### Specs
+
+Are you running the recommended Standard Hardware for Polkadot? Can you ensure that machines have
+enough processing power, memory, file storage, and network connectivity? It's helpful for nominators
+to know the specs of the machines a validator uses as to assess how they may perform in the network.
+If a validator is running underpowered machines, they may not want to nominate them, as these can
+result in less blocks produced and less overall rewards. In certain circumstances more powerful
+machines can result in higher rewards for both the Validator and their Nominators.
+
+#### Automation and orchestration approaches (terraform, ansible, chef, puppet, kubernetes, etc)
+
+What kind of approach is taken for spinning up and provisioning nodes? How might you automate
+spinning up large clusters of nodes and upgrading them? Elaborating on what kind of automation (or
+lack thereof) can help get a sense of how robust a validator setup is. There's many common actions
+or routine maintenance that needs to be done, and automating this type of thing often helps mitigate
+human errors.
+
+#### Network Topology
+
+Does the Validator node have protection against Denial of Service attacks, and if so how is that
+done? Outlining the kind of network topology of Validator infrastructure will help Nominators get a
+sense of how resilient operations are to attacks. Some things to highlight are usages of firewalls,
+VPNs, network segmentation, proxies, or other layers separation.
+
+#### Upgrading
+
+Both Polkadot and Kusama releases are published
+[here](https://github.com/paritytech/polkadot/releases). Validators are expected to upgrade their
+nodes as soon as a new release comes out. Although not every release is mandatory to upgrade to,
+each new release usually has bug fixes, optimizations, new features, or other beneficial changes.
+It's within the best interest of the entire network that validators update their nodes in a timely
+fashion. This signals to nominators that a validator is timely, cares about their operations, and is
+quick to adapt to necessary circumstances.
+
+It can also be useful for Nominators to know how the Validator runs software, and where they get new
+binaries from. How do they get alerted for new releases? Do they hear about things from matrix? Do
+they have alerts for particular Github activity? Do they use the Debian/RPM packages? Do they use
+the Parity provided Github binaries? Do they use Parity Docker images? Do they make their own Docker
+images? Do they build the binaries themselves? Validators often have their own build server for
+making binaries. If they take the extra steps to make these, and do not rely on external parties,
+this can be seen a plus from Nominators, as it helps contribute to decentralization.
+
+#### Logging, metrics, monitoring, and observability
+
```

Comment:
If possible, we could link the appropriate guides (i.e. secure validator, polkadot validator setup, etc...)